### ISSUE TITLE
[🐛 Bug]: Webdriver Bidi command timed out

### ISSUE THREAD

**Issue by levkovsb** [webdriverio/webdriverio#13474]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

latest

### Mode

WDIO Testrunner

### Which capabilities are you using?

```typescript
{
      maxInstances: 5,
      browserName: 'chrome',
      'goog:chromeOptions': {
        args: ['--headless', 'disable-gpu', '--log-level=3', '--window-size=1920,1080'],
      },
}
```

### What happened?

Our test timed out after 60 seconds because of the hardcoded timeout value in the current release. Path to the line: build/index.js, line 85. 

### What is your expected behavior?

Can you please provide an option to configure this timeout or increase it in your code so that the test cases that run longer than 60 seconds can still be finished?

### How to reproduce the bug.

Run any test case that runs longer than 60 seconds and the described error occurs

### Relevant log output

```typescript
Command script.callFunction with id 8 (with the following parameter: {"functionDeclaration":"function (...args) {\n            return new Promise(async (resolve, reject) => {\n                try {\n                    await function QunitFinishedEventInBrowserContext(done) {\n    QUnit.on('runEnd', function (qunitRunEnd) {\n        console.debug('QUnit runEnd event was triggered.'); // eslint-disable-line no-console\n        done(qunitRunEnd);\n    });\n    setTimeout(() => {\n        // Check whether tests have already been completed, QUnit event runEnd won't be triggered\n        if (QUnit.config?.started > 0 && QUnit.config?.queue?.length === 0) {\n            console.debug('QUnit.config.started:', QUnit.config?.started); // eslint-disable-line no-console\n            console.debug('QUnit.config.queue.length:', QUnit.config?.queue?.length); // eslint-disable-line no-console\n            console.debug('QUnit.config.pq.finished:', QUnit.config?.pq?.finished); // eslint-disable-line no-console\n            done();\n        }\n    }, 100);\n}.call(this, ...args, resolve)\n                } catch (err) {\n                    return reject(err)\n                }\n            })\n        }","awaitPromise":true,"arguments":[],"target":{"context":"CC074D3B6EC58E8D05B628C778C7847F"}}) timed out
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by ryan-cubic** [comment]
I'm also no longer able to run any test due to this BIDI related issue:

BiDi-CDP Mapper is controlling this tab
Closing or reloading it will stop the BiDi process. [Details.](https://github.com/GoogleChromeLabs/chromium-bidi)

cdp:SEND ▸ { "id": 0, "method": "Target.attachToBrowserTarget" }
cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "8F71DD68D74DDF9A1583042368040504", "targetInfo": { "targetId": "5e3e9824-ce91-4d17-9466-98034000a56e", "type": "browser", "title": "", "url": "", "attached": true, "canAccessOpener": false }, "waitingForDebugger": false } }
cdp:RECV ◂ { "id": 0, "result": { "sessionId": "8F71DD68D74DDF9A1583042368040504" } }
cdp:SEND ▸ { "id": 1, "method": "Target.getBrowserContexts", "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 2, "method": "Target.getTargets", "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 3, "method": "Security.setIgnoreCertificateErrors", "params": { "ignore": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 1, "result": { "browserContextIds": [] }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 2, "result": { "targetInfos": [ { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } ] }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 3, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 4, "method": "Target.setDiscoverTargets", "params": { "discover": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "method": "Target.targetCreated", "params": { "targetInfo": { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "method": "Target.targetCreated", "params": { "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 4, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 5, "method": "Target.setAutoAttach", "params": { "autoAttach": true, "waitForDebuggerOnStart": true, "flatten": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD", "targetInfo": { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 6, "method": "Page.enable", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:SEND ▸ { "id": 7, "method": "Page.getFrameTree", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:SEND ▸ { "id": 8, "method": "Runtime.enable", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:SEND ▸ { "id": 9, "method": "Page.setLifecycleEventsEnabled", "params": { "enabled": true }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:SEND ▸ { "id": 10, "method": "Target.setAutoAttach", "params": { "autoAttach": true, "waitForDebuggerOnStart": true, "flatten": true }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:SEND ▸ { "id": 11, "method": "Runtime.runIfWaitingForDebugger", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:SEND ▸ { "id": 12, "method": "Runtime.runIfWaitingForDebugger", "sessionId": "EB86692A668BC3D40630C0F6B680B757" }
cdp:RECV ◂ { "id": 5, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 6, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 7, "result": { "frameTree": { "frame": { "id": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "url": "about:blank", "domainAndRegistry": "", "securityOrigin": "://", "mimeType": "text/html", "adFrameStatus": { "adFrameType": "none" }, "secureContextType": "InsecureScheme", "crossOriginIsolatedContextType": "NotIsolated", "gatedAPIFeatures": [] } } }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Runtime.executionContextCreated", "params": { "context": { "id": 1, "origin": "://", "name": "", "uniqueId": "1780938485299442365.-4253671951501350087", "auxData": { "isDefault": true, "type": "default", "frameId": "2E74744E4EF603EC3D24A95729CCCE30" } } }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 8, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "commit", "timestamp": 9608.941063 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "DOMContentLoaded", "timestamp": 9608.941192 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "load", "timestamp": 9608.942472 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
debug:info Mapper instance has been launched
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkAlmostIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 9, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 10, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 11, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "id": 12, "result": {}, "sessionId": "EB86692A668BC3D40630C0F6B680B757" }
cdp:SEND ▸ { "id": 13, "method": "Target.detachFromTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "method": "Target.detachedFromTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9" }, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "id": 13, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" }
cdp:RECV ◂ { "method": "Page.frameResized", "params": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkAlmostIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }
cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }

**Comment by erwinheitzman** [comment]
Hi there, the path does not include what package it is in (I'm talking about the build/index.js) and without this info this cannot be reproduced.

Regarding your last message, it does not state an error nor do you describe what it going wrong. We need this kind of information in order to help out.

**Comment by erwinheitzman** [comment]
Tested a test longer than 60000ms and it runs just fine, don't forget to increase the timeout in your config that is specific for your framework.

**Comment by danielhjacobs** [comment]
The error message implied this comes from here:
https://github.com/webdriverio/webdriverio/blob/6e432db648bc5bc9b368e9bf64268a3cb7667d64/packages/webdriver/src/bidi/core.ts#L81-L84

**Comment by ryan-cubic** [comment]
> I'm also no longer able to run any test due to this BIDI related issue:
> 
> BiDi-CDP Mapper is controlling this tab Closing or reloading it will stop the BiDi process. [Details.](https://github.com/GoogleChromeLabs/chromium-bidi)
> 
> cdp:SEND ▸ { "id": 0, "method": "Target.attachToBrowserTarget" } cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "8F71DD68D74DDF9A1583042368040504", "targetInfo": { "targetId": "5e3e9824-ce91-4d17-9466-98034000a56e", "type": "browser", "title": "", "url": "", "attached": true, "canAccessOpener": false }, "waitingForDebugger": false } } cdp:RECV ◂ { "id": 0, "result": { "sessionId": "8F71DD68D74DDF9A1583042368040504" } } cdp:SEND ▸ { "id": 1, "method": "Target.getBrowserContexts", "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 2, "method": "Target.getTargets", "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 3, "method": "Security.setIgnoreCertificateErrors", "params": { "ignore": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 1, "result": { "browserContextIds": [] }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 2, "result": { "targetInfos": [ { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } ] }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 3, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 4, "method": "Target.setDiscoverTargets", "params": { "discover": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "method": "Target.targetCreated", "params": { "targetInfo": { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "method": "Target.targetCreated", "params": { "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" } }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 4, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 5, "method": "Target.setAutoAttach", "params": { "autoAttach": true, "waitForDebuggerOnStart": true, "flatten": true }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD", "targetInfo": { "targetId": "2E74744E4EF603EC3D24A95729CCCE30", "type": "page", "title": "about:blank", "url": "about:blank", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 6, "method": "Page.enable", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:SEND ▸ { "id": 7, "method": "Page.getFrameTree", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:SEND ▸ { "id": 8, "method": "Runtime.enable", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:SEND ▸ { "id": 9, "method": "Page.setLifecycleEventsEnabled", "params": { "enabled": true }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:SEND ▸ { "id": 10, "method": "Target.setAutoAttach", "params": { "autoAttach": true, "waitForDebuggerOnStart": true, "flatten": true }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:SEND ▸ { "id": 11, "method": "Runtime.runIfWaitingForDebugger", "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Target.attachedToTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:SEND ▸ { "id": 12, "method": "Runtime.runIfWaitingForDebugger", "sessionId": "EB86692A668BC3D40630C0F6B680B757" } cdp:RECV ◂ { "id": 5, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 6, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 7, "result": { "frameTree": { "frame": { "id": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "url": "about:blank", "domainAndRegistry": "", "securityOrigin": "://", "mimeType": "text/html", "adFrameStatus": { "adFrameType": "none" }, "secureContextType": "InsecureScheme", "crossOriginIsolatedContextType": "NotIsolated", "gatedAPIFeatures": [] } } }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Runtime.executionContextCreated", "params": { "context": { "id": 1, "origin": "://", "name": "", "uniqueId": "1780938485299442365.-4253671951501350087", "auxData": { "isDefault": true, "type": "default", "frameId": "2E74744E4EF603EC3D24A95729CCCE30" } } }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 8, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "commit", "timestamp": 9608.941063 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "DOMContentLoaded", "timestamp": 9608.941192 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "load", "timestamp": 9608.942472 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } debug:info Mapper instance has been launched cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkAlmostIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 9, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 10, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 11, "result": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "id": 12, "result": {}, "sessionId": "EB86692A668BC3D40630C0F6B680B757" } cdp:SEND ▸ { "id": 13, "method": "Target.detachFromTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetInfo": { "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9", "type": "page", "title": "BiDi-CDP Mapper", "url": "data:,", "attached": true, "canAccessOpener": false, "browserContextId": "700C6700FA4406468DF76F427C2F2B2D" }, "waitingForDebugger": false }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "method": "Target.detachedFromTarget", "params": { "sessionId": "EB86692A668BC3D40630C0F6B680B757", "targetId": "4B1FA7331BAEB37E2BF6C1019C3556A9" }, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "id": 13, "result": {}, "sessionId": "8F71DD68D74DDF9A1583042368040504" } cdp:RECV ◂ { "method": "Page.frameResized", "params": {}, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkAlmostIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" } cdp:RECV ◂ { "method": "Page.lifecycleEvent", "params": { "frameId": "2E74744E4EF603EC3D24A95729CCCE30", "loaderId": "E600E7137142598E2809BF0E8BEA9C6A", "name": "networkIdle", "timestamp": 9608.942241 }, "sessionId": "1D64202E2A2D2A64034C56A6F915CBDD" }

My test execution works again after downgrade from:
    "@wdio/cli": "^9.0.7",
    "@wdio/local-runner": "^9.0.7",

back to:
    "@wdio/cli": "^8.40.2",
    "@wdio/local-runner": "^8.40.2",

**Comment by siddharthmittal97** [comment]
Hi @erwinheitzman 

I also faced the Issue

Can you tell how to avoid this Bidi Mapper Tab Opening ?

**Comment by christian-bromann** [comment]
Can anyone provide a minimal reproduction case for this? I otherwise can only close this issue.

**Comment by levkovsb** [comment]
Here is the reproduction case. The instructions on how to reproduce it are docuemnted in README.md
[FIT-sample.zip](https://github.com/user-attachments/files/16907081/FIT-sample.zip)

**Comment by christian-bromann** [comment]
This bug should be fixed with the latest release `v9.1.1`. Please let me know if you still experience this issue. Thanks!
